src/app/parcels/add/AddParcelForm.tsx

```diff
@@ -88,7 +89,7 @@ const AddParcels = ({ clientId }: AddParcelProps): React.ReactElement => {
                         initialFields={initialParcelFields}
                         initialFormErrors={initialParcelFormErrors}
                         clientId={clientId}
-                        editMode={false}
+                        writeParcelInfoToDatabase={insertParcel}
```

Comment:
We have AddParcelForm, EditParcelForm that builds upon a common ParcelForm. I have refactored this such that all the add/edit specific code are in their respective AddParcelForm/EditParcelForm files, and the ParcelForm contains the common code. This means that we don't need the editMode boolean flag. Lmk if you agree with this idea/change :)

Reply:
I think this is good